text-generation-inference/server/text_generation_server/generator.py

```diff
@@ -188,7 +193,32 @@ def resume(self):
             self._mask = torch.cat([self._mask, torch.LongTensor([1])])
         self._state = Slot.State.READY
 
-    def append(self, next_token: int, next_token_text: str):
+    def _decode_next_tokens(
+        self,
+    ) -> str:
+        """Hack to hopefully support generate_stream for the maximum number of tokenizers"""
```

Comment:
😅 
can we test that?

Reply:
I tested it manually. I would need a bit of work to test individually some parts of the TGI server, without loading the whole thing. I will do that in an upcoming pull-request so that this goes into the release today.

Reply:
SGTM!